src/NuGet.Clients/NuGet.PackageManagement.UI/PackageLicenseUtilities.cs

```diff
@@ -119,6 +123,33 @@ internal static IReadOnlyList<IText> GenerateLicenseLinks(LicenseMetadata metada
             return list;
         }
 
+        private static Uri CreateEmbeddedLicenseUri(string packagePath, LicenseMetadata licenseMetadata)
+        {
+            Uri baseUri = Convert(packagePath);
+
+            var builder = new UriBuilder(baseUri)
+            {
+                Fragment = licenseMetadata.License
+            };
+
+            return builder.Uri;
+        }
+
+        /// <summary>
+        /// Convert a string to a URI safely. This will return null if there are errors.
+        /// </summary>
+        private static Uri Convert(string uri)
```

Comment:
Should this name be a bit more specific than just Convert? For example, ConvertPathToUri? While the types can be inferred from the arguments and return type, I find the name a bit generic. Alternatively, do we need this method at all since it's been used from a single location?

Reply:
@zivkan ^